packages/pages/src/bin/pages.ts

```diff
@@ -38,7 +34,41 @@ switch (command) {
     if (args.length == 2) {
       folderToCreate = args[1];
     }
-    init(folderToCreate || null);
+    await init(folderToCreate || null);
     break;
   }
+  case "generate": {
+    const generateTargets = ["features"];
+
+    if (args.length != 2) {
+      process.stdout.write(
+        `Missing a target to generate. Valid values are: ${generateTargets.join(
+          ", "
+        )}\n`
+      );
+      process.exit(1);
```

Comment:
return

Reply:
If the process exits there's no need to return.

Reply:
Yeah I know but seems safer to do an explicit return as well to prevent unintended behavior.

Reply:
Can't `A 'return' statement can only be used within a function body.`